### PR TITLE
Create ag-pssg.json

### PR DESCRIPTION
**Pull request by AubreyR**
## Summary
Example PR for content contribution

## Notable Changes 
added a carousel under topics
-
-

### CODE REVIEW

File: app-web/topicRegistry/ag-pssg.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "sourceProperties": {
+    "sources": [
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/AubreyR/test-markdown/",
+          "owner": "AubreyR",
+          "repo": "test-markdown",
+          "files": [
+            "README.md",
```
Comment (patricksimonian) [inline app-web/topicRegistry/ag-pssg.json:19]: no trailing commas in json :)

**Review by patricksimonian — CHANGES_REQUESTED** [review]

File: app-web/topicRegistry/ag-pssg.json (hunk 1 above)
Comment (patricksimonian) [inline app-web/topicRegistry/ag-pssg.json:19]:
```suggestion
            "README.md"
```

File: app-web/topicRegistry/ag-pssg.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "Product Owner"
+    ]
+  },
+  "sourceProperties": {
+    "sources": [
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/AubreyR/test-markdown/",
+          "owner": "AubreyR",
```
Comment (patricksimonian) [inline app-web/topicRegistry/ag-pssg.json:16]:
```suggestion
          "owner": "AubreyR",
          "branch": "main",
```

**Review by patricksimonian — CHANGES_REQUESTED** [review]
last change

**Comment by AubreyR** [comment]
Everything worked for my demo!  Closing this now!

**Pull request closed without merging**